classes/manager.php

```diff
@@ -102,6 +102,19 @@ public static function is_profiling(): bool {
                 );
     }
 
+    /**
+     * True if running cron or adhoc_task scripts.
+     *
+     * @return bool
+     */
+    public static function is_cron(): bool {
+        global $SCRIPT;
+        return CLI_SCRIPT && (
```

Comment:
It is certainly not recommended, but there is another way to run cron from the web:

/admin/cron.php